core/payment/src/api/payments.rs

```diff
@@ -70,7 +71,10 @@ async fn get_payments(
     };
 
     match listen_for_events(getter, timeout_secs).await {
-        Ok(payments) => response::ok(payments),
+        Ok(payments) => {
+            log::info!("[get_payments]: {}", payments.len());
```

Comment:
Please remove this log. It doesn't bring any value and is on much too high level.